### ISSUE TITLE
Wrong data in the order book list

### ISSUE THREAD

**Issue by ArturoJ** [slazarov/python-bittrex-websocket#47]
Sometimes it is often wrong to take the orders at the end of the book, it usually stays between 5 seconds to 5 minutes. The limitation of 10 orders is by Bittrex or by the module?

{'MarketName': 'BTC-ZCL', 'Nounce': 4408, 'Buys': 
[{'Quantity': 8.71577187, 'Rate': 0.00143}, 
{'Quantity': 128.0589916, 'Rate': 0.00142546}, 
{'Quantity': 396.69205873, 'Rate': 0.00142542}, 
{'Quantity': 150.57055717, 'Rate': 0.00142504}, 
{'Quantity': 140.62861093, 'Rate': 0.00142424}, 
{'Quantity': 2.0, 'Rate': 0.001424}, 
{'Quantity': 7.10991058, 'Rate': 0.00140298}, 
{'Quantity': 13.23308423, 'Rate': 0.00078},  **<-- Error**
{'Quantity': 1.93328652, 'Rate': 0.00077779},   **<-- Error**
{'Quantity': 2.0, 'Rate': 0.00077026}],   **<-- Error**

**Comment by slazarov** [comment]
Yes, it’s going to be removed in the next release. You can change the book depth, you have the documentation FYI.

**Comment by ArturoJ** [comment]
Perfect, thanks. 
I just changed it in auxiliary and do not change it in websocket_client. 
Are you thinking about adding the power to buy and sell?

**Comment by ArturoJ** [comment]
For the moment the bug seems resolved, make a request of 200 and create a second list that shows me 10, and for the moment everything is correct. I suppose that the bug is by limiting the list, that the information passes by and does not retain it in the requested place. Sorry for my English, I'm from Spanish.

**Comment by slazarov** [comment]
There is no need to change the code, you can set the depth when you subscribe. Set it to a large number.

**Comment by slazarov** [comment]
I will not add anything trading related. You can do that with https://github.com/ericsomdahl/python-bittrex

**Comment by ArturoJ** [comment]
THX
